.github/actions/setup-create-base64-config/action.yml

```diff
@@ -85,12 +53,6 @@ runs:
         selected_networks=$(convert_to_toml_array "$SELECTED_NETWORKS")
         log_targets=$(convert_to_toml_array "$LOGSTREAM_LOG_TARGETS")
 
-        if [ -n "$PYROSCOPE_SERVER" ]; then
-          pyroscope_enabled=true
-        else
-          pyroscope_enabled=false
-        fi
```

Comment:
why did you remove this logic? how will we decide now whether pyroscope should be enabled?